projects/media-viewer/src/lib/toolbar/side-bar/outline-view/outline-item/outline-item.component.ts

```diff
@@ -26,7 +26,7 @@ export class OutlineItemComponent implements OnInit {
 
   navigateLink() {
     if (this.outline.dest) {
-      this.linkService.navigateTo(this.outline.dest);
+      this.pdfJsWrapperFactory.navigateTo(this.outline.dest);
```

Comment:
if youre using pdfjswrapperfactory, you can get rid of the link service variable and initialise in ngoninit

Reply:
thanks for the comment, this has been updated